addons/mail/static/src/models/thread_view_topbar.js

```diff
@@ -64,13 +70,26 @@ registerModel({
             }
             await this.thread.toggleCall();
         },
+        /**
+         * @param {Event} ev
+         */
+        onClickShowCallSettingsMenu(ev) {
+            // FIXME maybe find another way to prevent both being open at the same time?
+            this.threadView.update({
+                isCallSettingsMenuOpen: true,
+                isMemberListOpened: false,
+            });
```

Comment:
Could be solved by introducing a "Sidebar" model and make it so that each handler set the content, and there's only 1 content at a time (so setting a new content replaces the previous one).

But I wouldn't introduce it now, as it feels overkill. It's fine with manually setting true/false for the moment.